src/lib/cache/subCache.cpp

```diff
@@ -33,6 +33,7 @@
 
```

Comment:
Thus, the "thus potentially improving search performance" part should be removed and maybe mentioning something about code cleaness or similar...